appdata.patch

```diff
@@ -5,7 +5,7 @@
    </description>
    <url type="homepage">https://audacious-media-player.org</url>
 +  <releases>
-+    <release date="2020-04-14" version="4.0.2"></release>
++    <release date="2020-06-01" version="4.0.4"></release>
```

Comment:
```suggestion
+    <release date="2020-06-01" version="4.0.4"></release>
+    <release date="2020-04-29" version="4.0.3"></release>
+    <release date="2020-04-14" version="4.0.2"></release>
```

Reply:
Seems like I can't apply this suggestion (due to permissions). Did you want to go ahead and apply this to this pull request, or did you want me to pull it into my tree and add it to the pull request?

Reply:
I can't apply this as well. :) Maybe because i am enabled beta GitHub features. Well, lets push this as is and try to fix this AppData with upstream for next release.